src/vivarium_census_prl_synth_pop/utilities.py

```diff
@@ -232,32 +232,42 @@ def random_integers(
         and which indexes the returned `pandas.Series`.
     randomness:
         RandomnessStream
+    additional_key:
+        A identifying key for the randomness stream (usually a string)
 
     Returns
     -------
     pandas.Series
         An indexed set of integers in the interval [a,b)
     """
-    return (randomness.get_draw(index=index) * max_val + min_val).round().astype(int)
+    return np.floor(randomness.get_draw(index=index, additional_key=additional_key) * max_val + min_val).astype(int)
```

Comment:
This still seems wrong. If you call `random_integers(min_val=5, max_val=10)`, you could draw a random number 0.99 and get `np.floor(0.99 * 10 + 5) == np.floor(14.9) == 14`, which is greater than the supposed max value of 10.